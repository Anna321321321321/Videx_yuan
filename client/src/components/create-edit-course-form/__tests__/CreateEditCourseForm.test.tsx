import { shallow } from 'enzyme';
import moment from 'moment';
import React from 'react';
import CreateCourseForm from '..';

describe('Create Edit Course Form Component Unit Tests', () => {
  it('Render component', () => {
    const wrapper = shallow(
      //@ts-ignore
      <CreateCourseForm
        name={{
          value: 'name'
        }}
        category={{
          value: 'default'
        }}
        releaseDate={{
          value: moment('1970-01-01 16:00Z')
        }}
        onChange={Function}
        onSubmit={Function}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
