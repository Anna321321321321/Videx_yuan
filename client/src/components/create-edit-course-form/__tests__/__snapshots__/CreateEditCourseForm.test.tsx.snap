// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Edit Course Form Component Unit Tests Render component 1`] = `
<CreateEditCourseForm
  category={
    Object {
      "value": "default",
    }
  }
  form={
    Object {
      "getFieldDecorator": [Function],
      "getFieldError": [Function],
      "getFieldInstance": [Function],
      "getFieldProps": [Function],
      "getFieldValue": [Function],
      "getFieldsError": [Function],
      "getFieldsValue": [Function],
      "isFieldTouched": [Function],
      "isFieldValidating": [Function],
      "isFieldsTouched": [Function],
      "isFieldsValidating": [Function],
      "isSubmitting": [Function],
      "resetFields": [Function],
      "setFields": [Function],
      "setFieldsInitialValue": [Function],
      "setFieldsValue": [Function],
      "submit": [Function],
      "validateFields": [Function],
      "validateFieldsAndScroll": [Function],
    }
  }
  name={
    Object {
      "value": "name",
    }
  }
  onChange={[Function]}
  onSubmit={[Function]}
  releaseDate={
    Object {
      "value": "1970-01-01T16:00:00.000Z",
    }
  }
/>
`;
